contracts/metagov/VlAuraDelegatorPCVDeposit.sol

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "../core/TribeRoles.sol";
+import "./DelegatorPCVDeposit.sol";
+
+interface IAuraLocker {
+    struct LockedBalance {
+        uint112 amount;
+        uint32 unlockTime;
+    }
+    struct EarnedData {
+        address token;
+        uint256 amount;
+    }
+
+    function lock(address _account, uint256 _amount) external;
+
+    function getReward(address _account, bool _stake) external;
+
+    function processExpiredLocks(bool _relock) external;
+
+    function emergencyWithdraw() external;
+
+    function delegates(address account) external view returns (address);
+
+    function getVotes(address account) external view returns (uint256);
+
+    function lockedBalances(address _user)
+        external
+        view
+        returns (
+            uint256 total,
+            uint256 unlockable,
+            uint256 locked,
+            LockedBalance[] memory lockData
+        );
+
+    function claimableRewards(address _account) external view returns (EarnedData[] memory userRewards);
+
+    function notifyRewardAmount(address _rewardsToken, uint256 _reward) external;
+}
+
+interface IAuraMerkleDrop {
+    function claim(
+        bytes32[] calldata _proof,
+        uint256 _amount,
+        bool _lock
+    ) external returns (bool);
+}
+
+/// @title Vote-locked AURA PCVDeposit
+/// @author eswak
+contract VlAuraDelegatorPCVDeposit is DelegatorPCVDeposit {
```

Comment:
should it not also be a snapshot delegator?

Reply:
0xMaha confirmed the delegation must happen onchain & not in the gnosis registry

> (me) what does voting with vlAURA look like ? Snapshot ?

> (maha) Snapshot yes (at least initially) but on chain delegation
> As opposed to delegating via the snapshot ui